File: src/main/java/no/nav/domain/pensjon/regler/gomregning/BeregningerTilFaktoromregningGrunnlag.java
package no.nav.domain.pensjon.regler.gomregning;

import no.nav.domain.pensjon.regler.beregning.Beregning;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class BeregningerTilFaktoromregningGrunnlag implements Serializable {

    
    private Long vedtakId;
    private List<Beregning> beregningGrunnlagListe;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/gomregning/BeregningerTilFaktoromregningGrunnlag.kt
package no.nav.pensjon.regler.domain.gomregning

import no.nav.pensjon.regler.domain.beregning.Beregning
import java.io.Serializable

class BeregningerTilFaktoromregningGrunnlag : Serializable {
    var vedtakId: Long? = null
    var beregningGrunnlagListe: MutableList<Beregning> = mutableListOf()
}